packages/store/internals/src/angular.ts

```diff
@@ -8,20 +9,12 @@ import { memoize } from './memoize';
 declare const ngDevMode: boolean;
 
 function _isAngularInTestMode(): boolean {
-  const platformRef: PlatformRef | null = getPlatform();
-  if (!platformRef) return false;
-  const compilerOptions = platformRef.injector.get(COMPILER_OPTIONS, null);
-  if (!compilerOptions) return false;
-  const isInTestMode = compilerOptions.some((item: CompilerOptions) => {
-    const providers = (item && item.providers) || [];
-    return providers.some((provider: any) => {
-      return (
-        (provider && provider.provide && provider.provide.name === 'MockNgModuleResolver') ||
-        false
-      );
-    });
-  });
-  return isInTestMode;
+  return (
+    typeof ɵglobal.__karma__ !== 'undefined' ||
+    typeof ɵglobal.jasmine !== 'undefined' ||
+    typeof ɵglobal.jest !== 'undefined' ||
+    typeof ɵglobal.Mocha !== 'undefined'
+  );
```

Comment:
What is the minimum version of Angular that it supports?

Reply:
It’s cross-version, because Angular has been using Karma as a test runner from the very beginning. Jest is available when using `@angular-builders/jest` and Nrwl Nx Jest runner. Mocha is also available since zone.js has a Mocha plugin. Otherwise no other frameworks are available for running tests.